doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,21 +117,25 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
 To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Application --serial-number <serial_number>
 
 You need to follow this flashing process only one time, as the PCB configuration will not change.
```

Comment:
What's the command to program the BICR in RoT? Is it any different from the previous one? @tjora

Reply:
The command would be identical, but you are using `--core Application` rather than `--core Secure`.

Note that I do not think we have done much testing on how to perform this replacement in ROT, so I am not sure its something we should recommend in the docs.

@David-Garcia-Polo suggested something like this, as the safe option (1):
1. Erase the existing BICR
2. Modify the board
3. Program the new BICR
4. Reset

Another one that would probably work (2):
1. Program the new BICR
2. Power of the device _before any resets_
3. Modify the board
4. Reset

If you do not need to modify the board, and just need to patch some values then this would be fine(3):
1. Program the new BICR
2. Reset

Strategy (1) would require that we test that sdsc will allow us to program a BICR if no BICR is programmed.
Strategy (2) would require that we have a lot of confidence that that are no undue resets between programming and powering of the board. Personally I think this is quite scary.
Strategy (3) should be fine, but I do not know how well this is tested by SysFW devs.